new/components/vcloud/update-vcloud-vm.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
```

Comment:
please move this script to the "manager" folder. We're trying to keep things as clean as possible. Anything that is related to the manager as an entity rather to one of the components should go under the manager folder.

Thanks